### PR TITLE
exercise6_for_review

### PR DESCRIPTION
**Pull request by rajatbansal93**

### CODE REVIEW

**Comment by rajatbansal93** [comment]
Commit message

File: javascript/exercise6/js/things.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+function Things(parent, color, movies, drink, bikes) {
```
Comment (rajatbansal93) [inline javascript/exercise6/js/things.js]: Naming issue

File: javascript/exercise6/js/things.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  this.parentList = parent;
+  this.childColor = color;
```
Comment (rajatbansal93) [inline javascript/exercise6/js/things.js]: naming issues

File: javascript/exercise6/js/things.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  this.childMovies= movies;
+  this.childDrink = drink;
+  this.childBikes = bikes;
+};
+
+Things.prototype.bindEvents = function() {
+  _this=this;
```
Comment (rajatbansal93) [inline javascript/exercise6/js/things.js]: spacing issue var _this = this;

File: javascript/exercise6/js/things.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  var index;
+    this.parentList[0].addEventListener("click", function() {
+      debugger;
```
Comment (rajatbansal93) [inline javascript/exercise6/js/things.js]: debugger

File: javascript/exercise6/js/things.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+      index=0;
```
Comment (rajatbansal93) [inline javascript/exercise6/js/things.js]: spacing issue

File: javascript/exercise6/js/things.js [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+      if(this.checked == true){
+        _this.subListVisibility(index);
+      }
+      else{
+        _this.subListHidden(index);
+      }
+    });
+    this.parentList[1].addEventListener("click", function() {
+      debugger;
```
Comment (rajatbansal93) [inline javascript/exercise6/js/things.js]: why debugger

**Comment by rajatbansal93** [comment]
please improve rejected

**Review by lord-sumit — APPROVED** [review]

**Pull request merged**